setup.cfg

```diff
@@ -39,7 +39,7 @@ install_requires =
     voluptuous>=0.8.10
     werkzeug
     trollius; python_version < '3.4'
-    tenacity>=4.6.0,<7.0.0
+    tenacity
```

Comment:
You might want to leave the min version, that should not change.

Reply:
I will fix this.